app/controllers/concerns/dcv/sites/configured_layouts.rb

```diff
@@ -1,13 +1,18 @@
 module Dcv::Sites::ConfiguredLayouts
+  def subsite_key
+    subsite_config['slug']
+  end
+
   def subsite_layout
     configured_layout = subsite_config['layout'] || 'default'
     configured_layout = DCV_CONFIG.fetch(:default_layout, 'portrait') if configured_layout == 'default'
+    configured_layout = subsite_key if configured_layout == 'custom'
```

Comment:
Since these are mutually exclusive, might be clearer to have an if/elsif.

Reply:
it's unlikely (why would you set the default layout to 'custom'?), but: because `configured_layout` is conditionally assigned in each of these lines, they aren't strictly exclusive. It might be clearer intent to change order of lines 8 and 9?

Reply:
Maybe I'm misunderstanding what you mean, but I was proposing this change:

```
if configured_layout == 'default'
  configured_layout = DCV_CONFIG.fetch(:default_layout, 'portrait')
elsif configured_layout == 'custom'
  configured_layout = subsite_key
end
```

The above code behaves the same way as the current code, but at a glance makes it clear that zero or one of the lines will run, but never both.  Skimming through the original code makes it seem like there may be some scenario where both could run, until you look at the trailing conditionals which show that it's not actually the case.  Just means a little more thinking as you read through.

I don't have really strong feelings about this though, so I'm okay if you'd prefer keeping as is.

Reply:
Right, what I was trying to say was *technically* both could run, but you would have had to configure the DLC default layout to be 'custom'. I was suggesting that a clearer version of the current would be:

```
    configured_layout = subsite_config['layout'] || 'default'
    configured_layout = subsite_key if configured_layout == 'custom'
    configured_layout = DCV_CONFIG.fetch(:default_layout, 'portrait') if configured_layout == 'default'
    configured_layout
```

but a hybrid of what we're both really saying might be best?:
```
    configured_layout = subsite_config['layout'] || 'default'
    return subsite_key if configured_layout == 'custom'
    return DCV_CONFIG.fetch(:default_layout, 'portrait') if configured_layout == 'default'
    configured_layout
```

Reply:
That works for me!  Thanks!

Reply:
Done!